.github/workflows/rust.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2022-04-24
+          toolchain: nightly
```

Comment:
Using the latest nightly has a tendency to break unexpectedly, so I would recommend pinning just to a newer nightly version.

Reply:
It actually looks like the async stack no longer requires nightly. Maybe we should just do an all-features build from stable and call it a day?

Reply:
If we can switch to stable, that would be great I think. :)

Reply:
If we can switch to stable, that would be great I think. :)

Reply:
Good point, if it works with stable that is better.